pages/index.js

```diff
@@ -3,10 +3,25 @@ import { GlobalContext } from "@/components/GlobalProvider";
 import { useContext } from "react";
 
 export default function HomePage() {
+
+  const stylesForLink = {
+    textDecoration: "none",
+    borderRadius: "5px",
+    padding: "5px 5px",
+    color: "black",
+    backgroundColor: '#e9e9ed',
+    fontSize: ".8rem"
+  }
+
+  const sectionWidth = {
+    widht: "100%",
+    textAlign: "center",
+  }
+
   const [globalState, setGlobalState] = useContext(GlobalContext)
   return (
-    <section>
-      <Link href="/create-plan">Create Plan</Link>
+    <section style={sectionWidth}>
+      <Link style={stylesForLink} href="/create-plan">Create Plan</Link>
```

Comment:
please don't use inline-style but styled-components instead.

If you are wondering how to style the Link component form Next.js, see [this handout](https://github.com/neuefische/hh-web-23-2/blob/main/sessions/react-styled-components/react-styled-components.md#styling-a-custom-component) as a reference

Reply:
I've been having too much fun, I guess.